00-HTML-CSS-basics/Punto 1/1.6/signUpForm.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+	<form>
+		<fieldset>
+			First Name: <input type="text" required><br>
+			Last Name: <input type="text" required><br>
+			Email: <input type="email" required><br>
```

Comment:
avoid the use of `<br>` tag as you are only using it with style purpose. Instead you can wrap each label, input pair with a `div` block.